log/log.go

```diff
@@ -0,0 +1,73 @@
+package log
+
+import (
+	"github.com/Sirupsen/logrus"
+	libmachinelog "github.com/docker/machine/libmachine/log"
```

Comment:
`libmachine/log` and `log` are two different packages.
In this proposal, libmachine doesn't preclude anything specific about logging (doesn't even depend on logrus), and drivers depend only on that.
On the other hand machine depends on logrus, and defines the logging strategy it uses (including bugsnag, etc).